### PR TITLE
docs: reimplement docs

### PR DESCRIPTION
**Pull request by SteKoe**

### CODE REVIEW

**Review by erikpetzold — COMMENTED** [review]
Are there changes in the content of the docs or only in the structure/generation?

**Comment by SteKoe** [comment]
Going to tidy up this PR!

File: spring-boot-admin-docs/src/site/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+$ USE_SSH=true yarn deploy
+```
+
+Not using SSH:
+
+```
+$ GIT_USER=<Your GitHub username> yarn deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
```
[the remark this answers is not in the document]
Comment (SteKoe) [reply spring-boot-admin-docs/src/site/README.md:41]: :)

File: spring-boot-admin-docs/src/site/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```
[the remark this answers is not in the document]
Comment (SteKoe) [reply spring-boot-admin-docs/src/site/README.md:8]: yes. it was just mentioned here in the readme. I am not a fan of missing package managers either, hence it is npm based.

**Comment by SteKoe** [comment]
Tidied up the MR #3908.

**Pull request closed without merging**